src/config/toDeploy.json

```diff
@@ -24,10 +11,7 @@
         "buySlopeDen": "36000000000000000000000",
         "investmentReserveBasisPoints": "1000",
         "revenueCommitmentBasisPoints": "1000",
-        "minInvestment": "1000000000000000",
-        "beneficiary": "0x519b70055af55A007110B4Ff99b0eA33071c720a",
```

Comment:
Why are `beneficiary`, `control', and `minInvestment` being removed?

Reply:
they appear removed from the unused DATinfoMainnet. But the final file will look like this:
```
{
    "DATinfo": {
        "collateralType": "ETH",
        "name": "Dxdao Token",
        "symbol": "DXD",
        "currency": "0x0000000000000000000000000000000000000000",
        "whitelistAddress": "0x0000000000000000000000000000000000000000",
        "initReserve": "100000000000000000000000",
        "initGoal": "4800000000000000000000",
        "buySlopeNum": "1",
        "buySlopeDen": "36000000000000000000000",
        "investmentReserveBasisPoints": "1000",
        "revenueCommitmentBasisPoints": "1000",
        "minInvestment": "100000000000000000",
        "feeBasisPoints": "0",
        "autoBurn": "true",
        "openUntilAtLeast": "0"
    }
}
```
minInvestment is not removed, and the control and beneficiary are set dynamically in the script, if not it uses the msg.sender

Reply:
whoops, meant to say `feeCollector` instead of `minInvestment` . .  Is feeCollector also set in the script?